test/parallel/test-tls-disable-renegotiation.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+// Tests that calling disableRenegotiation on a TLSSocket stops renegotiation.
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
```

Comment:
thinking about it, I think I need to go add the crypto skip in a couple of the other tests.